### PR TITLE
Upgrade Docker configuration for new version

### PR DESCRIPTION
**Pull request by SamuelHassine**

### CODE REVIEW

File: docker-compose-dev.yml [hunk 1]
```diff
@@ -27,7 +27,21 @@ services:
         hard: 65536
     ports:
       - 9200:9200
-      - 9300:9300
+      - 9300:minio
```
Comment (aya) [inline docker-compose-dev.yml]: typo ?

File: docker-compose.yml [hunk 2]
```diff
@@ -46,6 +61,11 @@ services:
       - REDIS__HOSTNAME=redis
       - REDIS__PORT=6379
       - ELASTICSEARCH__URL=http://elasticsearch:9200
+      - MINIO__ENDPOINT=minio
+      - MINIO__PORT=9000
+      - MINIO__USE_SSL=false
+      - MINIO__ACCESS_KEY=ChangeMeAccess
+      - MINIO__SECRET_KEY=ChangeMeKey
```
Comment (aya) [inline docker-compose.yml]: shouldn't be S3 instead of MINIO ?

File: docker-compose-dev.yml (hunk 1 above)
Comment (SamuelHassine) [reply docker-compose-dev.yml]: Typo, thx ;)

File: docker-compose.yml (hunk 2 above)
Comment (SamuelHassine) [reply docker-compose.yml]: For the moment we are not sure of the compatibility with S3, the configuration was named "Minio".

**Pull request merged**